BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -81,9 +81,15 @@ public struct AccessibilityElement: Element {
             config[\.accessibilityTraits] = accessibilityTraits
             config[\.isAccessibilityElement] = true
             config[\.accessibilityFrameSize] = accessibilityFrameSize
-            config[\.increment] = accessibilityIncrement
-            config[\.decrement] = accessibilityDecrement
             config[\.activate] = accessibilityActivate
+
+            guard let adjustable = traits.first(where: { $0 == .adjustable({}, {}) }),
```

Comment:
nit: I would use an `if` here since we don't really want an early return, it would be easy to add more configuration at the end of this closure without realizing it won't be executed

Reply:
changed to `if`